packages/react/src/components/Lightbox/__stories__/Lightbox.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, object, boolean } from '@storybook/addon-knobs';
+import '@carbon/grid/scss/grid.scss';
+import '../../../../../styles/scss/components/lightbox/_lightbox.scss';
```

Comment:
@sannminwin create an `index.scss` file within the story folder, which should then import these.

Reply:
updated with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/882/commits/afbd414592d1948f0861ff4866931e632f6389c6)